benchmarks/ekomark/benchmark/external/LHA_utils.py

```diff
@@ -62,10 +83,28 @@
     [   0,   0,   0,   0,   0,   0, 1/2,   0], # b
     [   0,   0,   0,   0,   0,   0,   0,   0], # t
 ])
+
+LHA_rotate_to_flavor_NNLO_FFNS = np.array([
+    # u_v, d_v, L_-, 2L_+, s_v  s_+, c_+,   g
+    [   0,   0,   0,   0,   0,   0,   0,   0], # ph
+    [   0,   0,   0,   0,   0,   0,   0,   0], # tbar
+    [   0,   0,   0,   0,   0,   0,   0,   0], # bbar
+    [   0,   0,   0,   0,   0,   0, 1/2,   0], # cbar
+    [   0,   0,   0,   0,-1/2, 1/2,   0,   0], # sbar
+    [   0,   0, 1/2, 1/4,   0,   0,   0,   0], # ubar
+    [   0,   0,-1/2, 1/4,   0,   0,   0,   0], # dbar
+    [   0,   0,   0,   0,   0,   0,   0,   1], # g
+    [   0,   1,-1/2, 1/4,   0,   0,   0,   0], # d
+    [   1,   0, 1/2, 1/4,   0,   0,   0,   0], # u
+    [   0,   0,   0,   0, 1/2, 1/2,   0,   0], # s
+    [   0,   0,   0,   0,   0,   0, 1/2,   0], # c
+    [   0,   0,   0,   0,   0,   0,   0,   0], # b
+    [   0,   0,   0,   0,   0,   0,   0,   0], # t
+])
 # fmt: on
```

Comment:
`eko.basis_rotation` contains the rotation from evolution to flavor basis already, thus there is room for dropping some code